profiler/src/profiler/TracyTimelineItemThread.cpp

```diff
@@ -275,6 +275,14 @@ void TimelineItemThread::DrawOverlay( const ImVec2& ul, const ImVec2& dr )
     m_view.DrawThreadOverlays( *m_thread, ul, dr );
 }
 
+void TimelineItemThread::DrawExtraPopupItems()
+{
+    if( m_view.GetViewData().drawCpuUsageGraph && ImGui::MenuItem( ICON_FA_TIMELINE " Select in CPU timeline" ) )
```

Comment:
That's not correct. You need to check if the cpu usage data exists. Looking at the queries the info window makes is a good starting point.